regolith/helpers/a_projectumhelper.py

```diff
@@ -366,3 +371,30 @@ def insert_checklists(self, pdoc, now):
             "status": "finished"
         }})
         return pdoc
+
+    # create Google Doc page for prum in Google Drive 'bg-projects'
+    def gdoc_generator(self):
```

Comment:
its ok here for now, but we might want to move this to tools or even a new module about external api's.  The code to do the calendar entry in a_presentation helper that I scrabbled together should be properly engineered too at this time, moved to tools.py too and hopefully much of the code reused.